hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/RatisUtil.java

```diff
@@ -100,9 +101,9 @@ private static void setRaftRpcProperties(final RaftProperties properties,
     Rpc.setRequestTimeout(properties, TimeDuration.valueOf(
         conf.getRatisRequestTimeout(), TimeUnit.MILLISECONDS));
     Rpc.setTimeoutMin(properties, TimeDuration.valueOf(
-        conf.getRatisRequestMinTimeout(), TimeUnit.MILLISECONDS));
```

Comment:
I think you don't need to update these parameters. The `cluster.waitForClusterToBeReady` is supposed to wait for SCM become leader thus solve the not leader exception.

Reply:
You are right, we only need the wait leader in cluster.waitForClusterToBeReady. The problem here is, we've set the min/max timeout un-properly. I will create a separate jira for his fix.